### PR TITLE
update log and logger phase scope.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @hellosnow @vicancy @ansyral @qinezh @superyyrrzz

### CODE REVIEW

File: src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs [hunk 1]
```diff
@@ -26,14 +21,13 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
             _originPhaseName = GetPhaseName();
             phaseName = _originPhaseName == null ? phaseName : _originPhaseName + "." + phaseName;
             SetPhaseName(phaseName);
-            if (enablePerformanceScope)
-            {
-                _performanceScope = new PerformanceScope("Scope:" + phaseName, LogLevel.Diagnostic);
-            }
         }
 
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: >LoggerPhaseScope(string phaseName, bool enablePerformanceScope) [](start = 15, length = 63)

if LoggerPhaseScope(phaseName, false), will still log
perfScope with Diagnostic?

File: src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs [hunk 2]
```diff
@@ -26,14 +21,13 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
...
+            : this(phaseName, LogLevel.Diagnostic) { }
+
         public LoggerPhaseScope(string phaseName, LogLevel perfLogLevel)
-            :this(phaseName, false)
```
Comment (928PJY) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: >: [](start = 12, length = 1)

space after colon?

File: src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs (hunk 1 above)
Comment (928PJY) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: maybe the same question: It seems like the boolean parameter do not play any roles, why don't we remove that?

---
In reply to: [95773945](https://github.com/dotnet/docfx/pull/1168#discussion_r95773945) [](ancestors = 95773945)

File: src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs [hunk 3]
```diff
@@ -26,14 +21,13 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
...
             phaseName = _originPhaseName == null ? phaseName : _originPhaseName + "." + phaseName;
             SetPhaseName(phaseName);
-            if (enablePerformanceScope)
-            {
-                _performanceScope = new PerformanceScope("Scope:" + phaseName, LogLevel.Diagnostic);
-            }
         }
 
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
+            : this(phaseName, LogLevel.Diagnostic) { }
```
Comment (928PJY) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: Question: This function seems to have never been referenced now, why should we still keep it?

Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: Obsolete first, then remove it.

---
In reply to: [95927157](https://github.com/dotnet/docfx/pull/1168#discussion_r95927157) [](ancestors = 95927157)

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

**Comment by 928PJY** [comment]
:shipit:

**Pull request merged**